tests/benchmark/stateful/bloatnet/depth_benchmarks/test_worst_case_attack_execute.py

```diff
@@ -0,0 +1,539 @@
+
+    # Compile with exact flags for reproducibility
+    result = run(
+        [
+            "solc",
+            "--bin",  # Get deployment bytecode (constructor + runtime)
+            "--optimize",
+            "--optimize-runs", "200",
+            "--metadata-hash", "none",  # CRITICAL: Exclude metadata for reproducible bytecode
+            str(sol_path),
+        ],
+        stdout=PIPE,
+        stderr=PIPE,
+        text=True,
+    )
+
+    if result.returncode != 0:
+        raise Exception(f"Failed to compile Solidity contract: {result.stderr}")
+
+    # Parse the output to get the deployment bytecode (includes constructor)
+    lines = result.stdout.split('\n')
+    bytecode_hex = None
+
+    # Look for the bytecode section
+    in_binary_section = False
+    for line in lines:
+        if "Binary:" in line:
+            in_binary_section = True
+        elif in_binary_section and line.strip() and not line.startswith("="):
+            bytecode_hex = line.strip()
+            break
+
+    if not bytecode_hex:
+        raise Exception(f"Could not extract bytecode from solc output:\n{result.stdout}")
+
+    # Remove 0x prefix if present
+    if bytecode_hex.startswith("0x"):
+        bytecode_hex = bytecode_hex[2:]
+
+    # Return raw bytes - we'll wrap it in Bytecode later with proper params
+    return bytes.fromhex(bytecode_hex)
+
+
+@pytest.mark.valid_from("Prague")
+@pytest.mark.parametrize("storage_depth,account_depth", [(9, 3)])  # Add more tuples as files become available
+def test_worst_depth_stateroot_recomp(
+    blockchain_test: BlockchainTestFiller,
+    pre: Alloc,
+    fork: Fork,
+    gas_benchmark_value: int,
+    storage_depth: int,
+    account_depth: int,
+) -> None:
+    """
+    BloatNet worst-case SSTORE attack benchmark with configurable depths.
+
+    Executes a worst-case attack pattern that maximizes trie traversal overhead:
+    1. Deploys 15,000 contracts via CREATE2 to addresses with configurable prefix sharing
+    2. Each contract contains storage slots at configurable trie depths
+    3. Generates optimized attack bytecode that calls attack() on multiple contracts
+    4. Respects gas limits: 16M per tx (Fusaka), 24KB contract size (EIP-170)
+    5. Verifies storage modifications using exact value tracking
+
+    Key optimizations:
+    - Function selector stored once and reused (saves 34 bytes per call)
+    - Batches attacks into multiple contracts when hitting size limits
+    - Uses 50,000 gas per CALL (sufficient for cold SSTORE operations)
+
+    Args:
+        storage_depth: Depth of storage slots in the contract (e.g., 10)
+        account_depth: Depth of account address prefix sharing (e.g., 5)
+    """
+    # Load pre-mined CREATE2 data for the given depth parameters
+    try:
+        create2_data = load_create2_data(storage_depth, account_depth)
+    except FileNotFoundError as e:
+        pytest.skip(f"Skipping test: {e}")
```

Comment:
This should be a failure. If we modify parametrization and add a bunch of invalid tests for which json files do not exist we will not notice as they will only show up as skipped tests.